Cart Flow/Custom Views/Buttons/CFButton.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CFButton.swift
+//  Cart Flow
+//
+//  Created by Scott Quintana on 8/6/20.
+//  Copyright © 2020 Scott Quintana. All rights reserved.
+//
+
+import UIKit
+
+class CFButton: UIButton {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        configure()
+    }
+    
+    
+    convenience init(backgroundColor: UIColor, title: String) {
+        self.init(frame: .zero)
+        self.backgroundColor = backgroundColor
+        self.setTitle(title, for: .normal)
+    }
+    
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
```

Comment:
Good to throw a `fatalError` here, but note that if your code is ever to be used in a Storyboard or XIB, you'll need to have this method implemented. It looks like you're setting up all of your UI in code for this app, so it won't matter, but if you wanted to reuse this class in another project that used Storyboards or XIBs, you'd have to come back and implement these when your brain isn't fresh on what needs to be done.